extensions/wikia/RTE/RTE.class.php

```diff
@@ -270,7 +270,25 @@ public static function makeGlobalVariablesScript( &$vars ) {
 			'maxage' => $wgSquidMaxage,
 		));
 
-		$vars['RTESiteCss'] = $wgServer . Skin::makeNSUrl( ( F::app()->checkSkin( 'oasis' ) ) ? 'Wikia.css' : 'Common.css', $query, NS_MEDIAWIKI );
+		$app = F::app();
+		$out = $app->wg->out;
+		$user = $app->wg->user;
+
+		if ( $app->checkSkin( 'oasis' ) ) {
+			$url = ResourceLoader::makeLoaderURL(
+				['site'],
+				$out->getLanguage()->getCode(),
+				$out->getSkin()->getSkinName(),
+				$user,
+				null,
+				ResourceLoader::inDebugMode(),
```

Comment:
Debug mode?

Reply:
Yeah, resource loader adds some additional debugging info if debug mode is on :)